core/pom.xml

```diff
@@ -162,7 +156,7 @@
                                         </goals>
                                     </pluginExecutionFilter>
                                     <action>
-                                        <ignore></ignore>
+                                        <ignore />
```

Comment:
This change reverts https://github.com/jdbi/jdbi/pull/1153

Reply:
yeah, but by fixing it at the core. Empty xml tags should in general be written as closed tags instead of open tags without content. Sortpom by default expands them, which is against that convention. Along with the change in sortpom's config elsewhere in this PR, this issue will be properly gone.

I'm re-doing all the work in this branch right now though, it's become a mess.